tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -388,15 +388,15 @@ def object_exists_return_func(input_s3_cli, input_glacier_bucket, input_file_key
             {request_files.INPUT_GRANULES_KEY: [expected_input_granule], request_files.INPUT_JOB_ID_KEY: job_id},
             result)
 
-    @patch('request_files.post_status_for_job_to_queue')
-    @patch('request_files.post_status_for_file_to_queue')
+    @patch('request_files.shared_recovery.create_status_for_job')
+    @patch('request_files.shared_recovery.update_status_for_file')
     @patch('time.sleep')
     @patch('request_files.restore_object')
     def test_process_granule_minimal_path(self,
```

Comment:
this test is failing with the error 

  `E           AssertionError: Expected call: create_status_for_job('6a0e838e-1db4-421f-82b4-648f150616b2', '82c2741e-f848-4cf0-a125-c2f85f38031f', '76fc5ddb-9047-4a18-b4e3-891c9e52f7c8', [{'filename': <ANY>}, {'key_path': <ANY>}, {'restore_destination': <ANY>}, {'status_id': 1}, {'error_message': None}, {'request_time': <ANY>}, {'last_update': <ANY>}, {'completion_time': None}], 'http://d572d790-592d-4dbb-96c8-67a84a11a3b2.blah')
E           
Actual call: create_status_for_job('6a0e838e-1db4-421f-82b4-648f150616b2', '82c2741e-f848-4cf0-a125-c2f85f38031f', '76fc5ddb-9047-4a18-b4e3-891c9e52f7c8', [{'filename': 'TBD'}, {'key_path': 'TBD'}, {'restore_destination': 'TBD'}, {'status_id': 1}, {'error_message': None}, {'request_time': '2021-05-27T16:42:55.093988+00:00'}, {'last_update': '2021-05-27T16:42:55.094000+00:00'}, {'completion_time': '2021-05-27T16:42:55.094004+00:00'}], 'http://d572d790-592d-4dbb-96c8-67a84a11a3b2.blah')`

Reply:
Judging from your screenshot, you're passing an int into update_status_for_file rather than an OrcaStatus.

Reply:
thanks. fixed